### DISCUSSION TITLE
Dynamically edit object returned by find()

### DISCUSSION THREAD

**Discussion by federicoodorizzi** [prisma/prisma#15979] [Q&A]
Hey, I'm not sure this is the right place to ask such question, but I’d like to dynamically edit the object returned by a Prisma query like so: 

```typescript
export const activities: QueryResolvers['activities'] = async () => {
  const activities = await db.activity.findMany({
    include: {
      author: true,
    },
  })

  activities.forEach((activity) => {
    activity.description = 'New description'
    activity.author.firstname = 'Rob'
  })

  return activities
}
```

This works fine with the first field, while it doesn’t seem to have any effect on the second one. 
Do you know if it possible to edit some fields of a nested object that was requested through an include?
I don't know if this is relevant, but I use graphql to send the result back to the client.

Thanks in advance!

**Comment by Rackar** [comment]
It should worked.  Debugger before activities.forEach() to check the raw array.

**Comment by nurul3101** [comment]
Hi @federicoodorizzi 👋 

Prisma returns plain [Javascript Objects](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/is-prisma-an-orm#conclusion) in responses so you should be able to mutate the response. Can you please provide a minimal reproduction so that I can check what could be the issue for this?

<img width="661" alt="Screenshot 2022-10-26 at 9 53 13 PM" src="https://user-images.githubusercontent.com/13049676/198081089-4852451c-e7b8-4eca-9b18-c95b0113ba7a.png">

**Reply by vladi-stevanovic** [reply]
:wave: Hello @federicoodorizzi , I’m going through pending GitHub Discussions to check if you need any help. Is this issue still open?

Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
